### DISCUSSION TITLE
request to add more IP reflector from APAC region

### DISCUSSION THREAD

**Discussion by malikshi** [sqm-autorate/sqm-autorate#91] [General] [answered]
this list ip from ID
```sh
27.50.20.21 moratelindo
202.162.214.250 iconplus
116.193.190.164 idch
103.56.148.179 beon
8.215.26.147 alibaba
103.47.134.195 my republic
```
can you guys maintainers check test and add those ip above as one of IP reflector, or if you guys had good IP from APAC it's would be great.

**Comment by Lochnair** [comment, marked as answer]
Confirmed all of them to be working, so created #92 to add these

**Reply by malikshi** [reply]
thank you very much
